Library/Formula/sfml.rb

```diff
@@ -27,9 +17,45 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Remove the bundled libs wording that causes linking errors.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  patch :DATA
```

Comment:
I think the one-line approving note of this idea in the other thread is about as supported as we're going to get here :crying_cat_face: